frontend_tests/node_tests/drafts.js

```diff
@@ -128,3 +129,18 @@ var draft_2 = {
     stub_draft({});
     assert.equal(drafts.snapshot_message(), undefined);
 }());
+
+(function test_initialize() {
+    var message_content = $("#new_message_content");
```

Comment:
This was going to be used to test `exports.update_draft`, but I don't know how to test it..